M2/configure.ac

```diff
@@ -559,8 +560,8 @@ fi
 AC_SUBST(GIT_DESCRIPTION)
 # TODO: currently finds the last commit that changed the VERSION file
 # but ideally it should get the last release commit instead
-NEW_COMMITS=`git rev-list $(git rev-list -1 HEAD $srcdir/VERSION)..HEAD --count`
-GIT_COMMIT=`git describe --dirty --always --match HEAD`
+NEW_COMMITS=`$GIT rev-list $($GIT rev-list -1 HEAD $srcdir/VERSION)..HEAD --count`
```

Comment:
wouldn't it now give "rev-list: command not found" errors?

Reply:
No, `$GIT` will be `false` if git isn't found, and `false rev-list ...` just silently fails.

Reply:
Ah, nice!